test-browser/tests/ballot.js

```diff
@@ -72,11 +72,17 @@ function runTests (browser, testData) {
         })
       })
       .perform((client, done) => {
-        console.log('addInstance 0x692a70d2e424a56d2c6c27aa97d1a86395877b3a')
+        console.log('addInstance invalid checksum address 0x692a70d2e424a56d2c6c27aa97d1a86395877b3a')
```

Comment:
that address `0x692a70d2e424a56d2c6c27aa97d1a86395877b3a` should not trigger `invalid check sum address` (there's only lowercase)

Reply:
@yann300 -- That address was just an example. We can also use the combination of uppercase and lowercase letters in order mimic an invalid checksum address